File: tests/packages/lsa/prog/lsalib.py
n = 0
m = 0
cnt = 0
finished = False


def init():
    global n, m
    n, m = map(int, input().split())


def guess(a):
    global finished, cnt
    cnt += 1

    if a == n:
        if cnt > m:
            print("Too_many_queries")
            print("WRONG")
            finished = True
            return 0
        else:
            print("Correct")
            print("OK")
            finished = True
            return 0
    if a < n:
        return -1
    if a > n:
        return 1


def quit():
    if not finished:
        print("Not_correct")
        print("WRONG")
    exit(0)

File: tests/packages/lsa/prog/lsalib.cpp
#include <bits/stdc++.h>
#include "lsalib.h"

using namespace std;

int n, m, cnt = 0;
bool finished = false;

void init() {
    cin >> n >> m;
}

int guess(int a) {
    cnt++;
    if (a == n) {
        if (cnt > m) {
            cout << "Too_many_queries" << endl;
            cout << "WRONG" << endl;
            finished = true;
            return 0;
        }
        else {
            cout << "Correct" << endl;
            cout << "OK" << endl;
            finished = true;
            return 0;
        }
    }
    if (a < n) {
        return -1;
    }
    return 1;
}

void quit() {
    if (!finished) {
        cout << "Not_correct" << endl;
        cout << "WRONG" << endl;
    }
    exit(0);
}
